orc8r/lib/go/registry/client/client_api.go

```diff
@@ -11,34 +11,20 @@
  * limitations under the License.
  */
 
-package service_registry
+package registry_client
```

Comment:
I don't love naming packages differently than their containing directory, you should just call this `client`. Maybe you can alias it to `registry_client` wherever you import it